cmd/edit/cluster/cmd.go

```diff
@@ -473,6 +484,99 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	/*******  AuditLogForwarding *******/
+	var auditLogRole *string
+	var auditLogRoleValue string
+	auditLogRoleExists := cluster.AWS().AuditLog().RoleArn() != ""
+
+	// user supplied value via flag in cli
+	if cmd.Flags().Changed("audit-log-arn") {
```

Comment:
Can we please extract this block to a new function? 
Something like `SetAuditLogForwarding`.